.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish package to npmjs
+
+on:
+  workflow_dispatch:
+    inputs:
+      bucketeer_version:
+       description: "Bucketeer version"
+       required: true
```

Comment:
I think this doesn't work as is.
We need to somehow update package.json's version property if we want to publish manually.